Formula/neovim.rb

```diff
@@ -0,0 +1,90 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io"
+
+  stable do
+    url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+    sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+
+    resource "luarocks" do
+      url "https://github.com/luarocks/luarocks.git",
+        :revision => "5d8a16526573b36d5b22aa74866120c998466697"
+    end
+  end
+
+  head do
+    url "https://github.com/neovim/neovim.git", :shallow => false
+
+    resource "luarocks" do
+      url "https://github.com/luarocks/luarocks/archive/2.4.2.tar.gz"
+      sha256 "eef88c2429c715a7beb921e4b1ba571dddb7c74a250fbb0d3cc0d4be7a5865d9"
+    end
+  end
+
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jemalloc" => :recommended
+  depends_on "libuv"
+  depends_on "msgpack"
+  depends_on "unibilium"
+  depends_on "libtermkey"
+  depends_on "libvterm"
+  depends_on "gettext"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+
+  resource "luv" do
+    url "https://github.com/luvit/luv/archive/1.9.1-0.tar.gz"
+    sha256 "86a199403856018cd8e5529c8527450c83664a3d36f52d5253cbe909ea6c5a06"
+  end
+
+  resource "luajit" do
+    url "https://github.com/LuaJIT/LuaJIT/archive/v2.0.4.tar.gz"
+    sha256 "d2abdf16bd3556c41c0aaedad76b6c227ca667be8350111d037a4c54fd43abad"
+  end
+
+  def install
+    resources.each do |r|
+      r.stage(buildpath/"deps-build/build/src/#{r.name}")
+    end
+
+    cd "deps-build" do
+      system "cmake", "../third-party",
+            "-DUSE_BUNDLED=OFF",
+            "-DUSE_BUNDLED_LUAROCKS=ON",
+            "-DUSE_BUNDLED_LUAJIT=ON",
```

Comment:
If I include the luajit formula and disable the above line I get this error when compiling
```
Looking for Lua...
luajit found in $PATH: /usr/local/opt/luajit/bin
Checking Lua interpreter...
luajit found in /usr/local/opt/luajit/bin
Checking Lua includes...
lua.h not found (looked in /usr/local/opt/luajit/include, /usr/local/opt/luajit/include/lua/5.1, /usr/local/opt/luajit/include/lua5.1)
```
The lua.h file is located in `/usr/local/opt/luajit/include/luajit-2.0` - can I force it somehow to search there ?